docker/hub/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM docker.io/golang:1.13.7-alpine3.11 AS builder
+
+RUN apk --update --no-cache add bash build-base
+
+WORKDIR /build
+
+ADD . /build
+
+RUN mkdir -p bin
+RUN sh ./build.sh
```

Comment:
I'd prefer we didn't do this as it makes it hard to know which `build.sh` will run (it depends where you run the dockerfile).

Reply:
Yeah - there's an unfortunate naming clash here too. Will think about that.

Reply:
So this is relative to the `WORKDIR`, which is `/build`.